app/assets/javascripts/availability.js.coffee

```diff
@@ -0,0 +1,29 @@
+jQuery ->
+  window.availability_updater = new AvailabilityUpdater
+class AvailabilityUpdater
+  constructor: ->
+    this.request_availability()
+  availability_url: "http://bibdata.princeton.edu/availability"
+  request_availability: ->
+    ids = this.record_ids().toArray()
+    params = $.param({ids: ids})
+    url = "#{@availability_url}?#{params}"
+    $.getJSON(url, this.process_records)
+  process_records: (records) =>
+    for record_id, availability_info of records
+      this.apply_record(record_id, availability_info)
+  apply_record: (record_id, availability_info) ->
+    return if availability_info.more_holdings == true
```

Comment:
Since the "more_holdings" field is supposed to mean that the user should click on the record to get full availability info, I'm thinking we could use the info icon (glyphicon glyphicon-info-sign) with a tooltip "Click on the record for full availability info" for this case.